ui/components/multichain/pages/page/components/header/header.tsx

```diff
@@ -9,13 +8,34 @@ import {
   TextVariant,
 } from '../../../../../../helpers/constants/design-system';
 
+import type { StyleUtilityProps } from '../../../../../component-library/box';
+
+interface HeaderProps extends StyleUtilityProps {
+  /**
+   * Elements that go in the page footer
+   */
+  children: React.ReactNode | React.ReactNode[];
+  /**
+   * Elements that go in the header end accessory
+   */
+  endAccessory?: React.ReactNode | React.ReactNode[];
+  /**
+   * Elements that go in the header start accessory
+   */
+  startAccessory?: React.ReactNode | React.ReactNode[];
+  /**
+   * Additional CSS class provided to the footer
+   */
+  className?: string;
+}
```

Comment:
I think we should create a separate types file for this

Reply:
Maybe we draw the limit at like 5?  I kinda loathe type files if the props are only used in one spot.